src/configs/best-practice.ts

```diff
@@ -8,6 +8,8 @@ const config = {
     'wc/no-closed-shadow-root': 'error',
     'wc/no-constructor-params': 'error',
     'wc/no-typos': 'error',
+    'wc/no-child-traversal-in-attributechangedcallback': 'error',
+    'wc/no-child-traversal-in-connectedcallback': 'error',
     'wc/require-listener-teardown': 'error'
```

Comment:
Would we not also consider the following as best practice?

- `wc/no-invalid-exends`
- `wc/no-customized-built-in-elements`

These are in the `recommended` config for github's plugin, as they represent errors in browsers without remediation in an individual codebase.

Reply:
you are right! i'll add them to best practice for now but it may be that we move them into recommended at some point (when we move a few others too)